src/main/java/Main.java

```diff
@@ -0,0 +1,8 @@
+public class Main {
+    public static void main(String[] args) {
+        ConsolePrinter consolePrinter = new ConsolePrinter();
+        int returnedIntVal = consolePrinter.printValue();
+        String formattedStrVal = String.format("~~~<%d>~~~", returnedIntVal);
+        System.out.println(formattedStrVal);
```

Comment:
Insted of this:
```
int returnedIntVal = consolePrinter.printValue();
String formattedStrVal = String.format("~~~<%d>~~~", returnedIntVal);
System.out.println(formattedStrVal);
```
Code could be simplified to one line:
`System.out.printf("~~~<%d>~~~%n", consolePrinter.printValue());`

Reply:
Done